cluster.yml

```diff
@@ -113,3 +113,8 @@
   roles:
     - { role: kubespray-defaults}
     - { role: kubernetes/preinstall, when: "dns_mode != 'none' and resolvconf_mode == 'host_resolvconf'", tags: resolvconf, dns_late: true }
+
+- hosts: k8s-cluster
+  roles:
+  - role: kubernetes/postinstall
```

Comment:
Do you want to run this on all nodes?

Reply:
Yeap, this is role for all nodes who have k8s components and need to rotate certs. But it,s not related to etcd because its certs will not expire during our live)